test/python/pulse/test_macros.py

```diff
@@ -91,6 +92,91 @@ def test_fail_measure(self):
         with self.assertRaises(PulseError):
             macros.measure(qubits=[0], inst_map=self.inst_map)
 
+    def test_measure_v2(self):
+        """Test macro - measure with backendV2."""
+        sched = macros.measure(qubits=[0], backend=self.backend_v2).filter(
+            channels=[MeasureChannel(0), AcquireChannel(0)]
+        )
+        expected = Schedule(
+            (
+                0,
+                Play(
+                    GaussianSquare(
```

Comment:
You cannot hard-code these values because backend calibration snapshot may be updated in future.

Reply:
I add a new test at https://github.com/Qiskit/qiskit-terra/pull/9987/commits/381a8e65d8a4586269ec59ca74a86a01a98661db.